citrination_client/views/search_template/client.py

```diff
@@ -10,5 +10,70 @@ def __init__(self, api_key, webserver_host="https://citrination.com", suppress_w
         members = ["create", "get_available_columns"]
         super(SearchTemplateClient, self).__init__(api_key, webserver_host, members, suppress_warnings, proxies)
 
-    def create(self, ml_template, search_template):
-        pass
+    def create(self, dataset_ids):
```

Comment:
I think you can just delete this and rename `create_with_extract_as_keys` to `create`